### PR TITLE
Fixes in saithrift to latest sai headers

### PR DESCRIPTION
**Pull request by ysegev**

### CODE REVIEW

File: test/saithrift/src/switch_sai.thrift [hunk 1]
```diff
@@ -187,12 +187,12 @@ service switch_sai_rpc {
     //vlan API
```
Comment (rivasiv) [inline test/saithrift/src/switch_sai.thrift]: Good changes. Do you regenerated this, or it's manually corrected?

Comment (ysegev) [inline test/saithrift/src/switch_sai.thrift]: Manually corrected according to new vlan header

File: test/saithrift/tests/sail3.py [hunk 2]
```diff
@@ -92,24 +92,26 @@ def runTest(self):
         rif_id2 = sai_thrift_create_router_interface(self.client, vr_id, 1, port2, 0, v4_enabled, v6_enabled, mac)
 
         addr_family = SAI_IP_ADDR_FAMILY_IPV4
-        ip_addr1 = '10.10.10.1'
+        ip_addr1 = '10.10.10.0'
         ip_mask1 = '255.255.255.0'
         dmac1 = '00:11:22:33:44:55'
         nhop_ip1 = '20.20.20.1'
+        ip_mask2 = '255.255.255.255'
```
Comment (lguohan) [inline test/saithrift/tests/sail3.py]: change /24 for all the directly reachable route?

File: test/saithrift/tests/sail2.py [hunk 3]
```diff
@@ -291,17 +293,13 @@ def runTest(self):
...
+        vlan_member3 = sai_thrift_create_vlan_member(self.client, vlan_id, lag_id1, SAI_VLAN_PORT_UNTAGGED)
+        vlan_member4 = sai_thrift_create_vlan_member(self.client, vlan_id, port4, SAI_VLAN_PORT_UNTAGGED)
+
 
         attr_value = sai_thrift_attribute_value_t(u16=vlan_id)
         attr = sai_thrift_attribute_t(id=SAI_PORT_ATTR_PORT_VLAN_ID, value=attr_value)
-        self.client.sai_thrift_set_port_attribute(port1, attr)
-        self.client.sai_thrift_set_port_attribute(port2, attr)
-        self.client.sai_thrift_set_port_attribute(port3, attr)
+        self.client.sai_thrift_set_port_attribute(lag_id1, attr)
```
Comment (lguohan) [inline test/saithrift/tests/sail2.py]: set port_vlan_id on the port before we move to the bridge port proposal?

File: test/saithrift/tests/sail3.py [hunk 4]
```diff
@@ -941,6 +956,7 @@ def runTest(self):
         addr_family = SAI_IP_ADDR_FAMILY_IPV4
         ip_addr1 = '10.10.0.0'
         ip_mask1 = '255.255.0.0'
+        ip_mask2 = '255.255.255.255'
```
Comment (lguohan) [inline test/saithrift/tests/sail3.py]: change to /24 for all direct reachable route. It's a more common use case. We are not yet to test corn cases here.

**Pull request merged**